src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -510,18 +530,21 @@ def test_WINNF_FT_S_PCR_2(self, config_filename):
   def generate_PCR_3_default_config(self, filename):
     """Generate the WinnForum configuration for PCR 3."""
     # File path where SAS UUT claimed ppa boundary generated in PCR.1 test
-    sas_uut_claimed_ppa_boundary_file_path = os.path.join('testcases', 'output',
-                                                          'test_WINNF_FT_S_PCR_1',
-                                                          'sas_uut_claimed_ppa_boundary_'
-                                                          'default.config.json')
+    pcr_1_test_config_file_path = os.path.join('testcases', 'configs',
+                                               'test_WINNF_FT_S_PCR_1',
+                                               'default.config')
+    pcr_1_test_config_file_name = pcr_1_test_config_file_path.split('/')[-1]
+    sas_uut_claimed_ppa_boundary_file_path = getSasUutClaimedPpaBoundaryFilePath(
+        pcr_1_test_config_file_name)
 
     # Load SAS UUT claimed ppa boundary and check if any error while retrieving
     # SAS UUT claimed ppa boundary generated in PCR.1 test.
     try:
       with open(sas_uut_claimed_ppa_boundary_file_path, 'r') as claimed_ppa_file:
         user_claimed_ppa_contour = json.load(claimed_ppa_file)
     except IOError:
-      raise RuntimeError('ConfigError:There is an error in reading path:%s'
+      raise RuntimeError('ConfigError:There is an error in reading path:%s \n'
```

Comment:
This error message would be useful in the body of the test case as well (since ITS will not use our default config generation function to create their configs).

Reply:
Agreed.   Bassed on the suggestion, This error message to indicate execute PCR 1 test   first to proceed the dependency tests and validate user claimed contour path   exist moved to corresponding test case method rather than reside in configure   method.